core/metrics/impl/prometheus/metrics_impl.cpp

```diff
@@ -0,0 +1,51 @@
+#include "metrics_impl.hpp"
```

Comment:
No license disclaimer at the beginning. You could change the default template for hpp and cpp files in your ide to make include the license header automatically

Reply:
good idea!